source/_integrations/axis.markdown

```diff
@@ -12,34 +12,69 @@ ha_iot_class: Local Push
 ha_codeowners:
   - '@Kane610'
 ha_domain: axis
+ha_qa_scale: platinum
 ---
 
 [Axis Communications](https://www.axis.com/) devices are surveillance cameras, speakers, access control and other security-related network connected hardware. Event API works with firmware 5.50 and newer.
 
-Home Assistant will automatically discover their presence on your network.
+Home Assistant will automatically discover their presence on your network. Discovery utilizes both DHCP, SSDP and Zeroconf protocols to discover devices.
 
 ## Configuration
 
 For configuration go to the `Integrations pane` on your Home Assistant instance.
 
 <div class='note'>
-  It is recommended that you create a user on your Axis device specifically for Home Assistant. For all sensor functionality, it is enough to create a user belonging to user group viewer. Light control requires a minimum of operator.
+  It is recommended that you create a user on your Axis device specifically for Home Assistant. For sensor functionality, it is enough to create a user with viewer privileges. If you want additional functional control you will need admin privileges.
 </div>
 
-## Troubleshooting discovery
+## Debugging integration
+
+If you have problems with a device or the integration you can add debug prints to the log.
+
+```yaml
+logger:
+  default: info
+  logs:
+    axis: debug
+    homeassistant.components.axis: debug
+```
+
+### Troubleshooting
+
+If you are having issues and want to report a problem, always start with making sure that you're on the latest [AXIS OS version](https://www.axis.com/support/firmware).
```

Comment:
```suggestion
If you are having issues and want to report a problem, always start with making sure that you're on the latest [Axis OS version](https://www.axis.com/support/firmware).
```

Reply:
This is a branding thing, can it be kept as is?